Example/Auth/Sample/MainViewController.m

```diff
@@ -2458,32 +2516,41 @@ - (void)signInWithPhoneNumber {
   }];
 }
 
+/** @fn signInWithPhoneNumberRecaptchaWithString
+    @brief Allows sign in with phone number using reCAPTCHA and completion.
+    @param phoneNumber Number pass in string for automation.
+ */
+- (void)signInWithPhoneNumberRecaptchaWithString:(NSString *_Nullable)phoneNumber
```

Comment:
I'd prefer the name signInWithPhoneNumber:(NSString *_Nullable)phoneNumber
                                                        completion:(void(^)(NSError *_Nullable))completion...

The reasons are:
- ReCAPTCHA is now the correct way to sign in with phone number so it is no longer a special case.
- We try out best to make sure the last part of the function name is the name of the actual variable (starting with a lowercase character);
Example:
`- (void) doSomethingWithSomethingElse:(NSString *)somethingElse;`

Notice that the SomethingElse at the end of the function name matches the "somethingelse" which is the variable name.

Same for other functions (update, link).